db/import-tmpl/map_drops.yml

```diff
@@ -0,0 +1,40 @@
+# This file is a part of rAthena.
+#   Copyright(C) 2022 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Map Drop Database
+###########################################################################
+#
+# Map Drop Settings
+#
+###########################################################################
+# - Map                             Name of the map
+#   GlobalDrops                     Drops for all monsters on this map. (Default: empty)
+#     - Item                        Item name.
+#       Rate                        Drop rate of item.
+#       RandomOptionGroup           Random Option Group applied to item on drop. (Default: None)
+#   SpecificDrops                   Drops for specific monsters on this map. (Default: empty)
+#     - Monster                     Monster name.
+#       Drops                       Drops for this specific monster. (Default: empty)
+#         - Item                    Item name.
+#           Rate                    Drop rate of item.
+#           RandomOptionGroup       Random Option Group applied to item on drop. (Default: None)
```

Comment:
Maybe add a line to give the specificities, or we need a doc
- Item unaffected by server drop rate
- Item can't be stolen from monster